src/index.ts

```diff
@@ -3,6 +3,8 @@ import senecaSn from './seneca-sn'
 import * as Seneca from 'seneca-await'
 import {startDispatch} from './dispatch'
 import cfg from './cfg'
+import {firebase} from './process-firebase'
```

Comment:
This feels dodgy, why not just create the firebase root and pass it on to the plugins that need it?  A DI-based approach instead of an import-based one.